lib/ansible/modules/network/ios/ios_l2_interface.py

```diff
@@ -459,10 +459,10 @@ def main():
                 module.fail_json(msg='You are trying to configure a VLAN'
                                  ' on an interface that\ndoes not exist on the '
                                  ' switch yet!', vlan=access_vlan)
-            elif native_vlan and native_vlan not in current_vlans:
-                module.fail_json(msg='You are trying to configure a VLAN'
-                                 ' on an interface that\ndoes not exist on the '
-                                 ' switch yet!', vlan=native_vlan)
+          #  elif native_vlan and native_vlan not in current_vlans:
```

Comment:
@geertn444 Thanks for the PR, can you please mention a one-liner comment as to why you have commented these lines for future reference.